### PR TITLE
[ELE-25] Responsivenesss

### PR DESCRIPTION
**Pull request by chaosua-del**
- Change text size on mobile devices (<sm)
- Change sidebar collapsibles 
- Change message view on the mobile devices (<sm)
- Add padding bottom for body and sidebar (<sm)

### CODE REVIEW

**Comment by Koriagami** [comment]
@chaosua-del All messages are left-aligned for Desktop. All good with Mobile

<details>
<summary>Screenshot</summary>

image


</details>

File: src/app/components/participants.tsx [hunk 1]
```diff
@@ -55,15 +55,18 @@ export default function Participants({ hubSlug }: ParticipantsProps) {
...
+                type="button"
+                className="w-full flex items-center justify-between gap-2 p-2 mr-1 lg:hover:text-white transition-colors"
+              >
+                <span>Participants</span>
                 <ChevronDown className="transition-transform group-data-[state=open]/collapsible:rotate-180" />
-              </Button>
+              </button>
             </CollapsibleTrigger>
 
             <Button
```
Comment (olehhapuk) [inline src/app/components/participants.tsx:72]:
```suggestion
          <CollapsibleTrigger className="grow flex items-center justify-between gap-2 py-2 pr-2 mr-1 lg:hover:text-white transition-colors">
            Participants
            <ChevronDown className="transition-transform group-data-[state=open]/collapsible:rotate-180 size-4" />
          </CollapsibleTrigger>
          <SidebarGroupAction>
```

Comment (olehhapuk) [reply src/app/components/participants.tsx:72]: Also please move collapsible trigger to a separate component, this was a good idea

File: src/app/components/rooms.tsx [hunk 2]
```diff
@@ -47,13 +47,16 @@ export default function Rooms({ hubSlug }: RoomsProps) {
...
-              <Button size="icon-sm" variant="ghost">
+              <button
+                type="button"
+                className="w-full flex items-center justify-between gap-2 p-2 mr-1 lg:hover:text-white transition-colors"
+              >
+                <span>Rooms</span>
                 <ChevronDown className="transition-transform group-data-[state=open]/collapsible:rotate-180" />
-              </Button>
+              </button>
             </CollapsibleTrigger>
```
Comment (olehhapuk) [inline src/app/components/rooms.tsx:60]: Same here

File: src/components/ui/sidebar.tsx [hunk 3]
```diff
@@ -200,7 +200,9 @@ const Sidebar = React.forwardRef<
           <SheetContent
             data-sidebar="sidebar"
             data-mobile="true"
-            className="w-[--sidebar-width] bg-sidebar p-0 text-sidebar-foreground [&>button]:hidden"
+            className={cn(
+              'w-[--sidebar-width] bg-sidebar p-0 text-sidebar-foreground [&>button]:hidden pb-5'
+            )}
```
Comment (olehhapuk) [inline src/components/ui/sidebar.tsx:205]: Why `cn` needed?

**Review by olehhapuk — CHANGES_REQUESTED** [review]

File: src/features/rooms/components/chat-message.tsx [hunk 4]
```diff
@@ -78,31 +78,37 @@ export default function ChatMessage({ message }: ChatMessageProps) {
   return (
     <div
       className={cn(
-        'flex self-start gap-2 max-w-[80%] bg-neutral-900 rounded-lg pl-2 pr-3 py-1',
-        isMine && 'self-end flex-row-reverse pl-3 pr-2'
+        'flex flex-col sm:flex-row self-start gap-2 max-w-[90%] bg-neutral-900 rounded-lg pl-2 pr-3 py-1',
+        isMine &&
+          'sm:flex-row-reverse lg:self-start self-end flex-col pl-3 pr-2 bg-gray-800'
```
Comment (olehhapuk) [inline src/features/rooms/components/chat-message.tsx:83]:
```suggestion
          'sm:flex-row-reverse 2xl:self-start self-end flex-col pl-3 pr-2 bg-neutral-800'
```

File: src/components/ui/collapsible.tsx [hunk 5]
```diff
@@ -1,9 +1,31 @@
...
+        className
+      )}
+      {...props}
+    >
+      {children}
+      <ChevronDown className="transition-transform group-data-[state=open]/collapsible:rotate-180 size-4" />
+    </CollapsibleTrigger>
+  );
+}
+
```
Comment (olehhapuk) [inline src/components/ui/collapsible.tsx:30]: This is specific to sidebar, please move to app/components/app-sidebar-dropdown-trigger.tsx or smth like this

**Review by olehhapuk — CHANGES_REQUESTED** [review]

**Review by olehhapuk — APPROVED** [review]

**Pull request merged**